api-docs/client-follow-up.md

```diff
@@ -0,0 +1,91 @@
+# Client follow up note
+
+The client follow up documents and updates any calls with the client and documents the interaction as well as set up a date to follow up with the client.
+
+## Creating a follow up note on a client
+
+### Request
```

Comment:
We should add a `**Notes**` section to the request, that indicates which fields are required or optional.